src/app/tracing/graph/graph.service.ts

```diff
@@ -509,8 +510,8 @@ export class GraphService {
         };
     }
 
-    private mapColor(color: number[]): string {
-        return `rgb(${color[0]},${color[1]},${color[2]})`;
+    private mapColor(color: Color): string {
```

Comment:
There is already a `colorToCss` utils function

Reply:
True, I've switched it over to using that one.

Reply:
There was also a colorToCss funciton in deliveries-properties.component.ts with identical code. I've switched that one to using the util function as well.